cypress/integration/show_work_orders.spec.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+describe('Show all work orders', () => {
+  it('Displays work order details, reference, date raised, last update, priority, property and description', () => {
+    //Stub request with work orders response
+    cy.login()
+    cy.server()
+    cy.fixture('work_orders/jobs.json').as('workorderslist')
+    cy.route('GET', 'api/repairs', '@workorderslist')
+    cy.visit(`${Cypress.env('HOST')}/repairs/jobs`)
+
+    cy.get('.govuk-table').within(() => {
+      cy.contains('th', 'Reference')
+      cy.contains('th', 'Date raised')
+      cy.contains('th', 'Last update')
+      cy.contains('th', 'Priority')
+      cy.contains('th', 'Property')
+      cy.contains('th', 'Description')
+    })
+
+    //can see reference number/ date raised, last updated, priority, property address, description
+    cy.get('.govuk-table__cell').within(() => {
+      cy.contains('00012345')
+      cy.contains('13 Jan 2021 4:24 pm')
+      cy.contains('14 Jan 2021 10:24 am')
```

Comment:
In the html it looks like the time is on another line so i'm not sure the spec would pick this up as one string